gui/default/syncthing/core/syncthingController.js

```diff
@@ -53,27 +53,30 @@ angular.module('syncthing.core')
         $scope.themes = [];
         $scope.globalChangeEvents = {};
         $scope.metricRates = false;
+        $scope.fsNotificationsAvailable = undefined;
```

Comment:
Isn't this in the most part simply an "introduction" of variables, as initialisations like `= []` and `= {}` are not necessary either? In which case all of these should be removed (that would however be a change unrelated to FS notifications).